### PR TITLE
Enterprise VA Sample with proactive notifications and multi-turn QnA

### PR DESCRIPTION
**Pull request by ryanisgrig**
### Purpose
*What is the context of this pull request? Why is it being done?*
#2407: Prepare Enterprise Assistant with VA Sample

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*
This uses the latest VA Sample with the following transposed:
* Proactive notifications code sample
* Multi-turn QnA dialog specifically for the HR FAQ

Clean up proactive notifications projects

### Tests
*Is this covered by existing tests or new ones? If no, why not?*
n/a

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*
Enterprise VA

#### General
- [x] I have commented my code, particularly in hard-to-understand areas	
- [x] I have updated related documentation

### CODE REVIEW

File: docs/_docs/solution-accelerators/tutorials/enable-proactive-notifications/4-virtual-assistant.md [hunk 1]
```diff
@@ -118,15 +118,75 @@ public class MainDialog : RouterDialog
...
++                activity.Recipient.Id = "1";
++                activity.Recipient.Name = "Bot";
++                activity.Recipient.Properties["role"] = "bot";
++            }
++        }
++    }
++
+    private class Events
+    {
+        public const string Location = "VA.Location";
```
Comment (lzc850612) [inline docs/_docs/solution-accelerators/tutorials/enable-proactive-notifications/4-virtual-assistant.md]: these are probably not necessary

File: samples/csharp/assistants/enterprise-assistant/EnterpriseAssistant.sln [hunk 2]
```diff
@@ -21,12 +21,12 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "WeatherSkill", "..\..\..\..
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ITSMSkill", "..\..\..\..\skills\csharp\experimental\itsmskill\ITSMSkill.csproj", "{E226ECDB-2D28-4515-95B5-5163BADDCBDA}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "VirtualAssistantSample", "VirtualAssistantSample\VirtualAssistantSample.csproj", "{0D14F534-36CE-4C95-B64A-18C5A78274F3}"
-EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EventHandler", "..\..\clients\event-handler.function-app\EventHandler.csproj", "{8258DB79-6B35-4F2D-8450-B3139B9B64FA}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EventProducer", "..\..\clients\event-producer.event-hub\EventProducer.csproj", "{F83F9D1F-8689-410F-9C84-4953E733EF16}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "VirtualAssistantSample", "VirtualAssistantSample\VirtualAssistantSample.csproj", "{BF9FBA28-C28B-4B15-81B1-C3E23D915551}"
```
Comment (lzc850612) [inline samples/csharp/assistants/enterprise-assistant/EnterpriseAssistant.sln]: this change is probably not necessary

Comment (lzc850612) [reply samples/csharp/assistants/enterprise-assistant/EnterpriseAssistant.sln]: change in this .sln file

File: samples/csharp/assistants/enterprise-assistant/VirtualAssistantSample/VirtualAssistantSample.csproj [hunk 3]
```diff
@@ -11,72 +11,69 @@
...
-    <PackageReference Include="Microsoft.Bot.Builder.LanguageGeneration" Version="4.6.0-Daily-2019-10-04-01" />
-    <PackageReference Include="Microsoft.Bot.Builder.LanguageGeneration.Templates" Version="4.6.0-Daily-2019-10-04-01" />
-    <PackageReference Include="Microsoft.Bot.Builder.Skills" Version="4.6.0-daily62" />
-    <PackageReference Include="Microsoft.Bot.Builder.Solutions" Version="4.6.0-daily62" />
-    <PackageReference Include="Microsoft.Bot.Builder.StreamingExtensions" Version="4.6.0-Daily-2019-09-25-01" />
-    <PackageReference Include="Microsoft.Bot.Builder.TemplateManager" Version="4.6.0-preview-191005-1" />
-    <PackageReference Include="Microsoft.Bot.Configuration" Version="4.6.0-preview-191005-1" />
-    <PackageReference Include="Microsoft.Bot.Connector" Version="4.6.0-preview-191005-1" />
-    <PackageReference Include="Microsoft.Bot.Schema" Version="4.6.0-preview-191005-1" />
+    <PackageReference Include="Microsoft.Bot.Builder" Version="4.6.0-rc1" />
```
Comment (lzc850612) [inline samples/csharp/assistants/enterprise-assistant/VirtualAssistantSample/VirtualAssistantSample.csproj]: we got 4.6.1 for bot builder sdk now. you should probably upgrade to that

**Review by lzc850612 — CHANGES_REQUESTED** [review]

File: samples/csharp/assistants/enterprise-assistant/VirtualAssistantSample/Services/DispatchLuis.cs [hunk 4]
```diff
@@ -1,5 +1,5 @@
 // <auto-generated>
-// Code generated by LUISGen
+// Code generated by LUISGen C:\Users\rylengel\source\repos\botframework-solutions\samples\csharp\assistants\enterprise-assistant\VirtualAssistantSample\Deployment\Scripts\..\Resources\Dispatch\en\bfenterpriseassistanten_Dispatch.json -cs Luis.DispatchLuis -o C:\Users\rylengel\source\repos\botframework-solutions\samples\csharp\assistants\enterprise-assistant\VirtualAssistantSample\Services
```
Comment (xieofxie) [inline samples/csharp/assistants/enterprise-assistant/VirtualAssistantSample/Services/DispatchLuis.cs]: personal info included.. same as GeneralLuis

Comment (ryanisgrig) [reply samples/csharp/assistants/enterprise-assistant/VirtualAssistantSample/Services/DispatchLuis.cs]: thanks!

**Review by lauren-mills — APPROVED** [review]

**Review by lzc850612 — APPROVED** [review]

**Pull request merged**